bin/common.sh

```diff
@@ -65,98 +65,6 @@ export_env_from_conf() {
     done < $1
 }
 
-# Read the config [mem_limit] from configuration file of BE and set the upper limit of TCMalloc memory allocation
-# if mem_limit is not set, use 90% of machine memory
-export_mem_limit_from_conf() {
```

Comment:
If the `export_mem_limit_from_conf`, how to set the mem limit when started?

Reply:
```
Status ExecEnv::init_mem_tracker() {
    int64_t bytes_limit = 0;
    std::stringstream ss;
    // --mem_limit="" means no memory limit
    bytes_limit = ParseUtil::parse_mem_spec(config::mem_limit, MemInfo::physical_mem());
    // use 90% of mem_limit as the soft mem limit of BE
    bytes_limit = bytes_limit * 0.9;
    if (bytes_limit <= 0) {
        ss << "Failed to parse mem limit from '" + config::mem_limit + "'.";
        return Status::InternalError(ss.str());
    }
```

can do this.

the script of common.sh only set env for tcmalloc.